src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/MicrosoftCodeAnalysisVisualStudioInteractiveComponents/Microsoft.CodeAnalysis.VisualStudioInteractiveComponents.swr

```diff
@@ -1,22 +0,0 @@
-use vs
```

Comment:
They wrapped out vsixes and produced manifest.json files that are used by setup authoring.  Now the VSSDK can produce these directly.